services/gps.md

```diff
@@ -0,0 +1,58 @@
+# GPS
+
+    identifier: 0x19dd6136
+    extends: _sensor
+    camel: gps
+
+A Global Positioning Device. NMEA message reference at https://www.sparkfun.com/datasheets/GPS/NMEA%20Reference%20Manual-Rev2.1-Dec07.pdf.
+
+## Registers
+
+    ro position @ reading {
+        timestamp: u32 utc_time
+        latitude: i12.20 ° { absolute_min=-90, absolute_max=90 }
```

Comment:
i9.23 would be enough for both (lat only needs i8.24 but let's keep it consistent)